app/maptools/recordingmaptool.cpp

```diff
@@ -45,8 +45,8 @@ void RecordingMapTool::addPoint( const QgsPoint &point )
 
     QgsPoint transformed = InputUtils::transformPoint(
                              PositionKit::positionCRS(),
-                             mFeatureLayerPair.layer()->sourceCrs(),
-                             mFeatureLayerPair.layer()->transformContext(),
+                             mActiveLayer->sourceCrs(),
```

Comment:
Missing sanity check for `mActiveLayer`  pointer (it can be null)
```suggestion
if ( mActiveLayer )
```

Reply:
Hmm, but we did not have this check here previously. Do we want to have such checks in every place where we access layer?

Reply:
Yeah, we need such checks everywhere, active layer can be null